pages/shoppingcart.js

```diff
@@ -0,0 +1,19 @@
+function ShoppingCart({ ingredients }) {
+  console.log(ingredients.filter((ingredient) => ingredient.chosen));
```

Comment:
The console.log should be removed before merging into main.

Reply:
done!